tests/test_pkce.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2016 Google Inc. All rights reserved.
```

Comment:
Shouldn't all the test modules be `_test_*`, since they're not part of the library-public API? /s